### PR TITLE
major: Refactor find and change commands\nBREAKING CHANGE: Replace dbMigrations and runDatabaseMigration with dbMigration. hUnified search across commands

### PR DESCRIPTION
**Pull request by iorg1**

### CODE REVIEW

File: operations/dbMigration.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+module.exports = (context, search) => {
+  findAndApply(context, search, (_, { name, path }) => {
+    let flags = ['-d'];
+    let itr = 0;
+    context.options.forEach(flag => {
+      if (flag.startsWith("-")) { flags[itr++] = flag }
+      else flags[1] = flag;
+    });
+
+    const cmd = `docker run -v ${path}:/app --network ${context.name} -e NODE_ENV=local --rm --name ${name}-setupdb ${name} npm run db-migration`;
```
Comment (jsaele) [inline operations/dbMigration.js:13]: Is it just me or isn't `flags` used? Also struggling a bit to understand this code. You're receiving a list of options that are pushed to an array if they start with a dash, if not they're explicitly added to index 1 of the array. 

So it's not a filter, but something else? Is it a flag key and value?

File: helpers/find.js [hunk 2]
```diff
@@ -21,8 +29,48 @@ const otherRepo = (context, search) => {
   return find(context.otherRepos, search);
 };
 
+const findAndApply = (context, search, func, options = {}) => {
+  if (search === 'all') {
+    if (options.onlyContainers) {
+      return context.containers.map((r) => func(context, r));
```
[the remark this answers is not in the document]
Comment (iorg1) [reply helpers/find.js:35]: r, short for repo.. :P

File: helpers/find.js [hunk 3]
```diff
@@ -21,8 +29,48 @@ const otherRepo = (context, search) => {
...
+    }
+  }
+
+  if (!options.onlyPackages && !options.onlyContainers) {
+    const other = find(context.otherRepos, search || getSearchFromCwd());
+    if (other) {
+      return func(context, other);
+    }
+  }
+  console.log(chalk`Could not find anything matching {blue "${search}"}`);
```
[the remark this answers is not in the document]
Comment (iorg1) [reply helpers/find.js:68]: y

File: helpers/find.js [hunk 4]
```diff
@@ -32,17 +32,17 @@ const otherRepo = (context, search) => {
 const findAndApply = (context, search, func, options = {}) => {
   if (search === 'all') {
     if (options.onlyContainers) {
-      return context.containers.map((r) => func(context, r));
+      return context.containers.map((repo) => func(context, repo));
```
Comment (jsaele) [inline helpers/find.js:35]: Very nice, `()` can be ditched though

File: operations/dbMigration.js [hunk 5]
```diff
@@ -5,10 +5,6 @@ module.exports = (context, search) => {
   findAndApply(context, search, (_, { name, path }) => {
     let flags = ['-d'];
     let itr = 0;
-    context.options.forEach(flag => {
-      if (flag.startsWith("-")) { flags[itr++] = flag }
-      else flags[1] = flag;
-    });
```
Comment (jsaele) [inline operations/dbMigration.js]: Shouldn't `flags` and `itr` be removed as well?

**Review by jsaele — APPROVED** [review]
🚀

**Review by kress92cs — APPROVED** [review]

**Pull request merged**